tests/integration/workspace_access/test_groups.py

```diff
@@ -199,14 +201,12 @@ def test_group_matching_names(
     ws, sql_backend, inventory_schema, make_random, make_user, make_group, make_acc_group, same_user
 ):
     rand_elem = make_random(4)
-    workspace_group_name = f"test_group_{rand_elem}"
-    account_group_name = f"same_group_[{rand_elem}]"
+    purge_suffix = get_purge_suffix()
+    workspace_group_name = f"test_group_{rand_elem}_{purge_suffix}"
+    account_group_name = f"same_group_[{rand_elem}]_{purge_suffix}"
     user1 = make_user()
     members1 = [user1.id]
-    members2 = [user1.id]
-    if not same_user:
-        user2 = make_user()
-        members2 = [user2.id]
```

Comment:
This change is here because the introduction of the `purge_suffix` variable above means the method had too many local variables for the linter. This was the easiest way to remove a local variable and avoid triggering the linter.